docs/resources/platform-security.md

```diff
@@ -25,7 +25,7 @@ Software Composition Analysis scans all the components used in an application fo
 
 ## Results
 
-<table><thead><tr><th width="242">Product</th><th data-type="number">SAST Score</th><th>DAST Score</th><th>Date</th><th data-hidden></th></tr></thead><tbody><tr><td>Subscription Manager</td><td>92</td><td>95</td><td>04 Jul 24</td><td></td></tr><tr><td>App Designer</td><td>94</td><td>95</td><td>04 Jul 24</td><td></td></tr><tr><td>Data Stream Designer</td><td>98</td><td>95</td><td>04 Jul 24</td><td></td></tr><tr><td>XMPro AI</td><td>100</td><td>Not Applicable</td><td>04 Jul 24</td><td></td></tr><tr><td>WorkFlow</td><td>100</td><td>91</td><td>05 Jul 24</td><td></td></tr></tbody></table>
+<table><thead><tr><th width="242">Product</th><th data-type="number">SAST Score</th><th>DAST Score</th><th>Date</th><th data-hidden></th></tr></thead><tbody><tr><td>Subscription Manager</td><td>92</td><td>95</td><td>27 Sep 24</td><td></td></tr><tr><td>App Designer</td><td>95</td><td>95</td><td>27 Sep 24</td><td></td></tr><tr><td>Data Stream Designer</td><td>99</td><td>95</td><td>27 Sep 24</td><td></td></tr><tr><td>XMPro AI</td><td>95</td><td>Not Applicable</td><td>27 Sep 24</td><td></td></tr></tbody></table>
```

Comment:
Hi Jonah, will this change not remove the Workflow line?
![image](https://github.com/user-attachments/assets/9f6fec14-df6f-478a-9ce5-27e8f0a4b8be)

Reply:
Yeh, this PR needs to be merged first https://github.com/XMPro/public-docs/pull/30